spec/controllers/groups_controller_spec.rb

```diff
@@ -17,6 +17,12 @@
       expect(::JSON.parse(response.body)['basic_group']['id']).to eq(group.id)
     end
 
+    it "responds with HTML" do
+      Guardian.any_instance.expects(:can_see?).with(group).returns(true)
```

Comment:
hmmm why is this needed? just create a visible group. the mocking here just risks instability long term